From andrea.gelmini@gmail.com  Fri Feb 26 16:40:37 2010
From: Andrea Gelmini <andrea.gelmini@gmail.com>
Date: Fri, 26 Feb 2010 10:15:09 +0100
Subject: Staging: comedi: dmm32at.c: Checkpatch cleanup
To: gregkh@suse.de
Cc: Andrea Gelmini <andrea.gelmini@gelma.net>
Message-ID: <1267175711-3586-16-git-send-email-andrea.gelmini@gmail.com>


From: Andrea Gelmini <andrea.gelmini@gelma.net>

WARNING: braces {} are not necessary for any arm of this statement
+	if (data[0] == COMEDI_OUTPUT) {
[...]
+	} else {
[...]

Signed-off-by: Andrea Gelmini <andrea.gelmini@gelma.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/comedi/drivers/dmm32at.c |    5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

--- a/drivers/staging/comedi/drivers/dmm32at.c
+++ b/drivers/staging/comedi/drivers/dmm32at.c
@@ -1048,11 +1048,10 @@ static int dmm32at_dio_insn_config(struc
 	 * value COMEDI_INPUT or COMEDI_OUTPUT. */
 
 	/* if output clear the bit, otherwise set it */
-	if (data[0] == COMEDI_OUTPUT) {
+	if (data[0] == COMEDI_OUTPUT)
 		devpriv->dio_config &= ~chanbit;
-	} else {
+	else
 		devpriv->dio_config |= chanbit;
-	}
 	/* get access to the DIO regs */
 	dmm_outb(dev, DMM32AT_CNTRL, DMM32AT_DIOACC);
 	/* set the DIO's to the new configuration setting */
